Makefile

```diff
@@ -21,8 +21,8 @@ dylibs:
 	# See https://github.com/calabash/calabash-codesign for details.
 	bin/make/dylibs.sh 337976ad9ace375ac06cd8fea2edb0c7276dec2a72d005ca5559a8bbf09c8841
 
-webquery_headers:
-	bundle exec bin/make/insert-js-into-webquery-headers.rb
+webquery_header:
```

Comment:
@jmoody In light of discovering that `set_text.js` is still used, I think it should actually be 'webquery_headers' :)